akka-actor/src/main/resources/reference.conf

```diff
@@ -420,6 +420,7 @@ akka {
         mailbox-type = "akka.dispatch.UnboundedMailbox"
       }
 
+      # deprecated (because relies on blocking)
```

Comment:
I don't think the purpose of the ticket was to deprecate all bounded mailboxes. Bounded mailbox can be good if used with zero `mailbox-push-timeout-time`.